src/lib/components/chat/Chat.svelte

```diff
@@ -179,6 +179,21 @@
 				} else {
 					message.citations = [data];
 				}
+			} else if (type === 'code_execution') {
```

Comment:
Perhaps it's better to use the existing 'citations' event type and add a new 'code_execution' sub data type here.

Reply:
Done.